driver-core/src/main/com/mongodb/client/model/search/ApproximateVectorSearchOptions.java

```diff
@@ -28,7 +28,7 @@
  *
  * @see Aggregates#vectorSearch(FieldSearchPath, Iterable, String, long, VectorSearchOptions)
  * @mongodb.atlas.manual atlas-vector-search/vector-search-stage/ $vectorSearch
- * @mongodb.server.release 6.0.10
+ * @mongodb.server.release 6.0.11, 7.0.2 or greater.
```

Comment:
We never say "or greater" when specifying `@mongodb.server.release`, let's not start doing it now to avoid confusion.

Reply:
I removed it from the Javadoc, but left it in the Scaladoc for consistency since we have used 'or greater' in the Scala documentation in the codebase, and it was present before this change.